app/controllers/application_controller.rb

```diff
@@ -1,5 +1,19 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  # def current_user
+  #   User.first
+  # end
+
```

Comment:
- [ ] Good job commenting the currrent_user method you can remove the commented code as it is not needed in your app and this will also make your code base more readable and neat👍🏿

Reply:
Done